classes/01class/exercises/c01-aws02/mail2kvanitha/README.md

```diff
@@ -0,0 +1,174 @@
+# C01-AWS02
```

Comment:
The answer is right and well done on that.
Just need some formatting to fix how the file will look like in the repo. Take a look:
https://github.com/devopsacademyau/academy/tree/mail2kvanitha/c01-aws02/classes/01class/exercises/c01-aws02/mail2kvanitha

Reply:
Formatting fixed.Please review